backend/src/services/expressService.ts

```diff
@@ -6,14 +6,19 @@ import playRouter from '../routers/playRouter.js';
 import queueRouter from '../routers/queueRouter.js';
 import searchRouter from '../routers/searchRouter.js';
 import spotifyRouter from '../routers/spotifyRouter.js';
-import { initWebsocket } from './websocketService.js';
+import { getPlayStatus } from './playService.js';
+import { emitEvent, initWebsocket } from './websocketService.js';
 
 export const createServer = async () => {
 	const app = express();
 	app.use(cors());
 	setupGlobalMiddleware(app);
 	setupRoutes(app);
 	const server = initWebsocket(app);
+	setInterval(async () => {
```

Comment:
I know this pr is already merged, but this introduces a bug, where the backend tries to call spotify before updating our token, leading to it crashing within 1 second of startup.

Reply:
...

Reply:
I tried it but that didn't happen? Maybe it was reusing the token from before?

Reply:
But yes, this should probably be fixed

Reply:
A very simple fix is to just check the auth token before each access, but that's not a great solution...